core/spec.md

```diff
@@ -3189,6 +3191,50 @@ and the following Resource level attributes:
     be silently ignored. See [Registry APIs](#registry-apis) for more
     information.
 
+##### `compatibility` Attribute
+- Type: String (with a defined enumeration of possible values)
+- Description: indicates whether Versions of this Resource need to adhere to
+  a certain compatibility rule. For example, a "backward" compatibility
+  value would indicate that all Versions of a Resource are mandated to be
+  backwards compatible with the next oldest Version, as determined by their
+  `createdat` timestamp attributes.
+
+  The exact meaning of what each `compatibility` value means might vary based
+  on the data model of the Resource, therefore this specification only defines
+  a very high-level abstract meaning for each to ensure some degree of
+  consistency. However, domain specification specifications are expected to
```

Comment:
Nit: "domain specification specifications"

Reply:
it's always amuses me what my fingers type out of habit. I still can't type "contain" w/o first typing "container" and having to hit backspace twice. :-)